data/sql_updates/create_omnibus_dates.sql

```diff
@@ -0,0 +1,163 @@
+-- Trying to make the names flow together as best as possible
+-- To keep the titles consice states are abbreviated as multi state if there is more than one
+-- Like:
+    -- House General Election FL
+    -- DEM Convention NH
+    -- General Elecion Multi-state
+create or replace function generate_election_title(trc_election_type_id text, office_sought text, election_states text[], party text)
+returns text as $$
+    begin
+        return case
+        when array_length(election_states, 1) > 1 then array_to_string(
+            array[
```

Comment:
Lines 11-17 are very similar to lines 18-24, and only differ in the last element of the array. Can we express with less repetition?